src/ewatercycle/observation/caravan.py

```diff
@@ -56,7 +55,8 @@ def get_caravan_data(
             lon         float64 8B ...
             area        float64 8B ...
             streamflow  (time) float32 12B ...
-        Attributes:
+
+    Attributes:
```

Comment:
```suggestion
        Attributes:
```
This is the output of xarray, but is misidentified by the formatter? It should stay indented.

Reply:
This was probably result of `ruff check --fix --unsafe-fixes`

To fix I had to noqa [E501](https://docs.astral.sh/ruff/rules/line-too-long/), [D410](https://docs.astral.sh/ruff/rules/no-blank-line-after-section/), [D411](https://docs.astral.sh/ruff/rules/no-blank-line-before-section/) and [D214](https://docs.astral.sh/ruff/rules/section-not-over-indented/)